easybuild/easyconfigs/i/ImageJ/ImageJ-1.5.1i.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PackedBinary'
+
+name = 'ImageJ'
+version = '1.5.1i'
+
+homepage = 'https://imagej.nih.gov/ij'
+description = "Image Processing and Analysis in Java"
```

Comment:
No, triple-quoting is only needed if newlines appear in the string.

Also this is not a docstring, it's just a regular string variable.

Docstrings do not need to be triple-quoted either strictly speaking, but they usually are since they tend to include one or more newlines.